core/lib/node_framework_derive/tests/ui.rs

```diff
@@ -0,0 +1,11 @@
+#[test]
+fn ui_pass() {
```

Comment:
Bikeshedding: Could make sense to place UI tests in the `node_framework` crate to test that they work with real traits. Also, wouldn't require to keep mock definitions of these traits for every test, which could potentially get outdated.

Reply:
My concern here is that AFAIU `trybuild` needs to build & link every test, and `zksync_node_framework` currently depends on most of our dependency tree, so it would be pretty slow.
It certainly makes sense to move tests there, but maybe it would be better to do once we move the implementations out of the framework crate (going to happen right after we rework `WiringLayer` & finish EN support)? I can create a task for that.

Reply:
Yup, the first run is very slow, but re-runs are fast. I guess we can live with it for a while.
Moved it there.